README.md

```diff
@@ -39,6 +39,20 @@ Places to ask questions and find other community resources.
 - [EU Working Group "APIs for Innovative Public Services (API4IPS)"](https://ec.europa.eu/cefdigital/wiki/pages/viewpage.action?pageId=254313406)
   - [REST best practices](https://ec.europa.eu/cefdigital/wiki/pages/viewpage.action?pageId=335282435) by that working group
 
+## Governmental and federal ambitions to establish more OSS
+Projects, working groups and policy papers concerning OSS in public administration.
+
+### Germany
+- Deutsche Cloud Strategie / cloud strategy germany
+  https://www.cio.bund.de/Web/DE/Innovative-Vorhaben/Deutsche-Verwaltungscloud-Strategie/deutsche_verwaltungscloud_strategie_node.html
+- Zentrum für digitale Souveränität / digital sovereignity centre
+  https://www.cio.bund.de/Web/DE/Innovative-Vorhaben/ZenDiS/zendis_node.html
```

Comment:
There's a lot of controversy about ZenDiS, especially about their potential impact. Some further reading would be nice